features/polyExplorer/src/components/clusterStory/clusterStory.css

```diff
@@ -60,6 +60,7 @@
 }
 
 .cluster-story-title {
+    color: var(--story-color-primary) !important;
```

Comment:
The important normally indicates that there's something else that shouldn't be there... What would that be?

Reply:
Styling being done with an `*`. Has highest prio and can only be overwritten with important